### PR TITLE
Version pump to v1.3.0

### PR DESCRIPTION
**Pull request by jothepro**
@a4z do you know if there is a way to test the changes before merging? Is it possible to add a tap from a branch?

### CODE REVIEW

**Review by a4z — APPROVED** [review]
my way of testing it is:
I set the tap url to my fork, update my fork and install it from there, 
if everything works, then I create the RP

**Comment by wcrestfield** [comment]
@jothepro Is there anything holding this back?

**Comment by jothepro** [comment]
Omg no, I totally forgot about this PR. 🙈 
I'd like to test the change like described by @a4z before merging, I'll do that asap and then merge it!

Thx for asking!

**Pull request merged**
